js/localization/cs.json

```diff
@@ -0,0 +1,239 @@
+        "dxDataGrid-summaryAvg": "Prům.: {0}",
+        "dxDataGrid-summaryAvgOtherColumn": "Průměr ze {1} je {0}",
+        "dxDataGrid-summarySum": "Suma: {0}",
+        "dxDataGrid-summarySumOtherColumn": "Suma {1} je {0}",
+        "dxDataGrid-summaryCount": "Počet: {0}",
+        "dxDataGrid-columnFixingFix": "Uchytit",
+        "dxDataGrid-columnFixingUnfix": "Uvolnit",
+        "dxDataGrid-columnFixingLeftPosition": "Vlevo",
+        "dxDataGrid-columnFixingRightPosition": "Vpravo",
+        "dxDataGrid-exportTo": "Export",
+        "dxDataGrid-exportToExcel": "Export do sešitu Excel",
+        "dxDataGrid-excelFormat": "soubor Excel",
+        "dxDataGrid-selectedRows": "Vybrané řádky",
+        "dxDataGrid-exportSelectedRows": "Export vybraných řádků",
+        "dxDataGrid-exportAll": "Exportovat všechny záznamy",
+        "dxDataGrid-headerFilterEmptyValue": "(prázdné)",
+        "dxDataGrid-headerFilterOK": "OK",
+        "dxDataGrid-headerFilterCancel": "Zrušit",
+        "dxDataGrid-ariaColumn": "Sloupec",
+        "dxDataGrid-ariaValue": "Hodnota",
+        "dxDataGrid-ariaFilterCell": "Filtrovat buňku",
+        "dxDataGrid-ariaCollapse": "Sbalit",
+        "dxDataGrid-ariaExpand": "Rozbalit",
+        "dxDataGrid-ariaDataGrid": "Datová mřížka",
+        "dxDataGrid-ariaSearchInGrid": "Hledat v datové mřížce",
+        "dxDataGrid-ariaSelectAll": "Vybrat vše",
+        "dxDataGrid-ariaSelectRow": "Vybrat řádek",
+
+        "dxTreeList-ariaTreeList": "Tree list",
+        "dxTreeList-editingAddRowToNode": "Přidat",
+
+        "dxPager-infoText": "Strana {0} ze {1} ({2} položek)",
+        "dxPager-pagesCountText": "ze",
+
+        "dxPivotGrid-grandTotal": "Celkem",
+        "dxPivotGrid-total": "{0} Celkem",
+        "dxPivotGrid-fieldChooserTitle": "Výběr pole",
+        "dxPivotGrid-showFieldChooser": "Zobrazit výběr pole",
+        "dxPivotGrid-expandAll": "Rozbalit vše",
+        "dxPivotGrid-collapseAll": "Sbalit vše",
+        "dxPivotGrid-sortColumnBySummary": "Srovnat \"{0}\" podle tohoto sloupce",
+        "dxPivotGrid-sortRowBySummary": "Srovnat \"{0}\" podle tohoto řádku",
+        "dxPivotGrid-removeAllSorting": "Odstranit veškeré třídění",
+        "dxPivotGrid-dataNotAvailable": "nedostupné",
+        "dxPivotGrid-rowFields": "Pole řádků",
+        "dxPivotGrid-columnFields": "Pole sloupců",
+        "dxPivotGrid-dataFields": "Pole dat",
+        "dxPivotGrid-filterFields": "Filtrovat pole",
+        "dxPivotGrid-allFields": "Všechna pole",
+        "dxPivotGrid-columnFieldArea": "Zde vložte pole sloupců",
+        "dxPivotGrid-dataFieldArea": "Zde vložte pole dat",
+        "dxPivotGrid-rowFieldArea": "Zde vložte pole řádků",
+        "dxPivotGrid-filterFieldArea": "Zde vložte filtr pole",
+
+        "dxScheduler-editorLabelTitle": "Předmět",
+        "dxScheduler-editorLabelStartDate": "Počáteční datum",
+        "dxScheduler-editorLabelEndDate": "Konecné datum",
```

Comment:
Koncové datum

Reply:
@BingoRUS the rest looks OK. So i already finished the review. Only those 2 changes are needed. BTW we don't have the term for "tree list" i believe

Reply:
@jdvjdv82 Thanks, I applied your suggestions and going to merge Czech localization now.